### PR TITLE
It's time for Main Page

### PR DESCRIPTION
**Pull request by UladBohdan**
The changes are:
- Orange color is replaced with Turquoise.
- Made tiles on Main page smaller: 4 tiles in the first row instead of three.
- Tiles are moved into a separate file and are included with a parameterized include-s. Hope that helps frontend development.
- Background is removed completely.
- Photos for authors (optional as well as author name is).
- Footer on login page.
- Turquoise transparent logo on Main page.
- Some font changes.
- Copyright 2017 -> 2018 😆 
- Some minor changes I don't remember about.

screencapture-localhost-3001-main-page-main-page-html-1514922534082

### CODE REVIEW

**Comment by Drapegnik** [comment]
like for extracting tile as component!

**Comment by UladBohdan** [comment]
@Drapegnik I knew you'll like it!

**Comment by Drapegnik** [comment]
I think button `load more` can be more interesting / complex

**Comment by UladBohdan** [comment]
I'd postpone any `load more` button changes till the moment somebody has an inspiration to modify it. We can add your request to trello / github issues.

File: src/tiles/tile.ejs [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<!-- List of variables used below. This suppresses lint warnings. -->
```
[the remark this answers is not in the document]
Comment (UladBohdan) [reply src/tiles/tile.ejs]: @tyndria thanks for your css remarks!

**Pull request merged**